CedarJava/src/main/java/com/cedarpolicy/BasicAuthorizationEngine.java

```diff
@@ -100,6 +117,40 @@ private static <REQ, RESP> RESP call(String operation, Class<RESP> responseClass
         }
     }
 
+    /**
+     * The result of processing an EntityValidationRequest.
+     */
+    @JsonIgnoreProperties({"result"})  // Ignore only the 'result' field
+    private static final class EntityValidationResponse {
```

Comment:
We usually put these types of definitions in a separate file (like you did for `EntityValidationRequest`)

Reply:
I can extract this class, but I’m wondering whether it should be public or package-private. Should the package clients have access to this class? Currently, when clients use the entityValidation function, it returns void and throws a BadRequestException if the entities are not valid.

What do you suggest?

Reply:
Seems like it could be package-private since the structure is just to aid deserialization.

Reply:
Ok I will implement the necessary changes.